docs/contributing/releasing.md

```diff
@@ -3,26 +3,46 @@
 > :memo: The versioning scheme we use is [SemVer](http://semver.org/). Note that until
 > we agree we're ready for v1.0.0, we will not increment the major version.
 
-1. Ensure all desired features are merged to `main` branch and `CHANGELOG.md` is updated.
+1. Ensure all desired features are merged to `main` branch and `CHANGELOG.md` is
+   updated. **Do not** edit the `[Unreleased]` header in the CHANGELOG -- the next step
+   will automatically update it.
 1. Use `bump-my-version` to increase the version number in all needed places, e.g. to
```

Comment:
We should include the files that `bump-my-version` changes which is: CHANGELOG.md, pyproject.toml, and CITATION.cff

Reply:
I'm not sure how I feel about that. I could be convinced either way. The other side, in my mind, is then we have to keep that list in sync when we change the config. Perhaps something along the lines of "The files which will be edited by bump-my-version are configured in `pyproject.toml`"?

Reply:
That makes sense. I like the idea of pointing folks to a list of files that will be edited (`pyproject.toml`) as it also provides some context for the `bump-my-version` command.

Reply:
3004088 What do you think?

Reply:
Perfect!